src/gazebo/gazebo_files/worlds/competition_field.sdf

```diff
@@ -31,7 +31,76 @@
 
     <include>
       <uri>package://ros_gz_description/models/master_ASM</uri>
-      <pose>3 -0.4 1 0 0 0</pose>
+      <pose>3 1 1 0 0 0</pose>
+    </include>
+
+    <!-- <model name="camera">
```

Comment:
@pwmarshall Why is there a big comment left here?

Reply:
This is still here